docs/sources/alerting/monitor/_index.md

```diff
@@ -1,27 +1,166 @@
 ---
+aliases:
+  - ./meta-monitoring/ # /docs/grafana/<GRAFANA_VERSION>/alerting/meta-monitoring/
+  - ./set-up/meta-monitoring/ # /docs/grafana/<GRAFANA_VERSION>/alerting/set-up/meta-monitoring/
 canonical: https://grafana.com/docs/grafana/latest/alerting/monitor/
-description: Monitor alerting metrics and data
+description: Meta monitoring
 keywords:
   - grafana
-  - alert
-  - monitoring
+  - alerting
+  - meta-monitoring
 labels:
   products:
-    - cloud
     - enterprise
     - oss
 menuTitle: Monitor
-title: Monitor alerting metrics
-weight: 180
+title: Meta monitoring
+weight: 200
 ---
 
-## Monitor alerting metrics
+# Meta monitoring
 
 Monitor your alerting metrics to ensure you identify potential issues before they become critical.
 
-[Meta monitoring][meta-monitoring]
+Meta monitoring is the process of monitoring your monitoring system and alerting when your monitoring is not working as it should.
 
-{{% docs/reference %}}
-[meta-monitoring]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/monitor/meta-monitoring"
-[meta-monitoring]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/monitor/meta-monitoring"
-{{% /docs/reference %}}
+In order to enable you to meta monitor, Grafana provides predefined metrics.
+
+Identify which metrics are critical to your monitoring system (i.e. Grafana) and then set up how you want to monitor them.
+
+You can use meta-monitoring metrics to understand the health of your alerting system in the following ways:
+
+1. [Optional] Create a dashboard in Grafana that uses this metric in a panel (just like you would for any other kind of metric).
+2. [Optional] Create an alert rule in Grafana that checks this metric regularly (just like you would do for any other kind of alert rule).
+3. [Optional] Use the Explore module in Grafana.
```

Comment:
To do what?

Reply:
You can use meta-monitoring metrics to understand the health of your alerting system in the following ways: